### PR TITLE
Nerf withered insomniacs, add eye markings as penance

### PR DESCRIPTION
**Pull request by Recoherent**
## About The Pull Request

makes the eyebags from the All Nighter quirk far less visible, and adds standard markings for the undereye region they occupy and the outer eye pixel on human/anthro characters.

new eyebags
image
new eyebags with outereye added for cohesion on human characters
image
undereye added to replicate how dark the previous eyebags were
image

## Why It's Good For The Game

quirks with mechanically enforced visual changes are fine and all but boy howdy are these visible by default. the markings let you more easily visually work around them as well. it's character customization, baby

## Changelog

:cl:
image: The eyebags from the All Nighter quirk are now far less blatant.
add: New 'undereye' head markings, in case you want to keep your eyebags as dark as they were before.
add: New 'outer eye' head markings for all your anime human eye or multicolored anthro eye needs.
/:cl:

### CODE REVIEW

**Comment by Recoherent** [comment]
wait no something broke with the actual files being committed what on earth

**Comment by Recoherent** [comment]
or did it? i am wildly confused. this should only change four files, and definitely not the bodypart dmi in the non-modular area. i am bad at git

**Comment by Recoherent** [comment]
got it set

File: code/datums/bodypart_overlays/simple_bodypart_overlay.dm [hunk 1]
```diff
@@ -27,6 +27,9 @@
 
 ///bags drawn beneath the eyes
 /datum/bodypart_overlay/simple/bags
+	//doppler - redirect to overwrite to make these less dark
+	icon = 'modular_doppler/modular_quirks/overwrites/icons/bodypart_overlay_simple.dmi'
```
Comment (Paxilmaniac) [inline code/datums/bodypart_overlays/simple_bodypart_overlay.dm:31]: document it like this
icon = 'modular_doppler/modular_quirks/overwrites/icons/bodypart_overlay_simple.dmi' // DOPPLER EDIT - whatever you want to say here
remove the comment on line 30 when you do that
also remove the single new line you added under line 31

**Review by Paxilmaniac — CHANGES_REQUESTED** [review]

File: code/datums/bodypart_overlays/simple_bodypart_overlay.dm (hunk 1 above)
Comment (Recoherent) [reply code/datums/bodypart_overlays/simple_bodypart_overlay.dm]: set

**Review by Paxilmaniac — APPROVED** [review]

**Pull request merged**